docs/modules/ROOT/pages/documentation/ecosystem/maven-plugin.adoc

```diff
@@ -243,6 +243,9 @@ values. key-value pairs will be formatted as `key=value`.
 |_none_
 |Defines a list of command line options that will be passed onto payara-micro.
 
+|`hotDeploy`
+|false
+|Enables the hot reload mode.
```

Comment:
```suggestion
|Enables the Hot Reload mode.
```
Or shouldn't we say Hot Deploy instead? The name of the feature is Hot Deploy and not Hot Reload, right? If it's the same feature as described in https://blog.payara.fish/hot-deploy-feature-in-payara-platform-5.201